src/bun.js/bindings/BunObject.cpp

```diff
@@ -22,6 +22,9 @@
 #include "BunObject.h"
 #include "WebCoreJSBuiltins.h"
 #include <JavaScriptCore/JSObject.h>
+#include <cmath>
+#include <cstddef>
+#include <sys/param.h>
```

Comment:
- Doesn't exist on Windows
- using `std::min` is more idiomatic than the `MIN` macro

```suggestion

```